### DISCUSSION TITLE
WooCommerce integration?

### DISCUSSION THREAD

**Discussion by franwzb** [flyntwp/flynt#447] [Q&A] [answered]
Hello! 👋

I'm been using Flynt and it's awesome!

Do you have plans to integrate with WooCommerce?

**Comment by timohubois** [comment, marked as answer]
@franwzb thank you for the question. Flynt is a starter theme and it provide a starting point for development. Regarding to this behavior, a implementation of WooCommerce inside the core of Flynt is not planned, currently.

There was a [previous discussion](https://github.com/flyntwp/flynt/discussions/358#discussioncomment-1706418) where you eventually find help to integrate WooCommerce by your own.
